packages/manager/apps/web/client/app/domain/zone/domain-zone-dns.html

```diff
@@ -1,4 +1,4 @@
-<div class="container-fluid px-0">
+<div class="container-fluid px-0" ui-view>
```

Comment:
```suggestion
<div class="container-fluid px-0" data-ui-view>
```

because `div` is not a component 🙂